org.eclipse.languageserver/src/org/eclipse/languageserver/languages/csharp/CSharpKeywordsHighlighter.java

```diff
@@ -32,121 +30,29 @@
 
 public class CSharpKeywordsHighlighter extends PresentationReconciler {
 
-	private Set<String> keywords = new HashSet<>(Arrays.asList(new String[] {
-		"abstract", 
-		"as",
-		"base",
-		"bool",
-		"break",
-		"byte",
-		"case",
-		"catch",
-		"char",
-		"checked",
-		"class",
-		"const",
-		"continue",
-		"decimal",
-		"default",
-		"delegate",
-		"do",
-		"double",
-		"else",
-		"enum",
-		"event",
-		"explicit",
-		"extern",
-		"false",
-		"finally",
-		"fixed",
-		"float",
-		"for",
-		"foreach",
-		"goto",
-		"if",
-		"implicit",
-		"in",
-		"int",
-		"interface",
-		"internal",
-		"is",
-		"lock",
-		"long",
-		"namespace",
-		"new",
-		"null",
-		"object",
-		"operator",
-		"out",
-		"override",
-		"params",
-		"private",
-		"protected",
-		"public",
-		"readonly",
-		"ref",
-		"return",
-		"sbyte",
-		"sealed",
-		"short",
-		"sizeof",
-		"stackalloc",
-		"static",
-		"string",
-		"struct",
-		"switch",
-		"this",
-		"throw",
-		"true",
-		"try",
-		"typeof",
-		"uint",
-		"ulong",
-		"unchecked",
-		"unsafe",
-		"ushort",
-		"using",
-		"virtual",
-		"void",
-		"volatile",
-		"while",
-	}));
-	
-	private Set<String> contextualKeywords = new HashSet<>(Arrays.asList(new String[] {
-		"add",
-		"alias",
-		"ascending",
-		"async",
-		"await",
-		"descending",
-		"dynamic",
-		"from",
-		"get",
-		"global",
-		"group",
-		"into",
-		"join",
-		"let",
-		"orderby",
-		"partial",
-		"remove",
-		"select",
-		"set",
-		"value",
-		"var",
-		"where",
-		"yield",
-	}));
+	private Set<String> keywords = new HashSet<>(Arrays.asList(new String[] { "abstract", "as", "base", "bool", "break",
```

Comment:
This is exactly where an automatic formatter shows it doesn't really know how to communicate with humans ;)